packages/contracts/src/predeploys.ts

```diff
@@ -19,5 +19,6 @@ export const predeploys = {
   OVM_ProxyEOA: '0x4200000000000000000000000000000000000009',
   OVM_ExecutionManagerWrapper: '0x420000000000000000000000000000000000000B',
   OVM_GasPriceOracle: '0x420000000000000000000000000000000000000F',
+  OVM_SequencerFeeVault: '0x4200000000000000000000000000000000000011',
```

Comment:
Did you intentionally skip `0x4200000000000000000000000000000000000010` ?

Reply:
i'll take it if y'all wont xDD

Reply:
Yes was intentionally skipped because @maurelian was using it in his safety cache PR